package.json

```diff
@@ -48,6 +48,7 @@
     "lodash": "^4.17.4",
     "memdown": "^1.4.1",
     "multihashes": "~0.4.12",
+    "multihashing-async": "^0.4.7",
```

Comment:
Was this dep necessary?

Reply:
yes, `test/blockstore-test` requires it.  Its a devDependency